tools/props/Versions.props

```
@@ -18,7 +18,7 @@
   <!-- NetFx and NetCore project dependencies -->
   <PropertyGroup>
     <AzureIdentityVersion>1.3.0</AzureIdentityVersion>
-    <MicrosoftIdentityClientVersion>4.22.0</MicrosoftIdentityClientVersion>
+    <MicrosoftIdentityClientVersion>4.47.2</MicrosoftIdentityClientVersion>
```

Revert. Not part of the fix.

```suggestion
    <MicrosoftIdentityClientVersion>4.22.0</MicrosoftIdentityClientVersion>
```